sdk/storage/AzureStorageBlob/Source/StorageBlobClient.swift

```diff
@@ -63,6 +64,12 @@ public final class StorageBlobClient: PipelineClient {
 
     private static let manager = URLSessionTransferManager.shared
 
+    internal static let viewContext: NSManagedObjectContext = manager.persistentContainer.viewContext
+
+    internal static func newBackgroundContext() -> NSManagedObjectContext {
```

Comment:
Doesn't look like this is ever called?

Reply:
It isn't currently called because once I enabled concurrency debugging to begin using this it was clear that should be a separate PR.

Reply:
I've removed it for now.